pkg/salt-api.service

```diff
@@ -4,8 +4,8 @@ Documentation=man:salt-api(1) file:///usr/share/doc/salt/html/contents.html http
 After=network.target
 
 [Service]
-Type=notify
```

Comment:
I just realized that we should split this commit. On the past we removed the `Type=notify` and put `Type=simple' but we removed that few months ago, so according to this, this commit should only set `User=salt` here.